tests/test_schema_editor_partitioning.py

```diff
@@ -275,3 +275,73 @@ def test_schema_editor_add_default_partition(method, key):
     schema_editor.delete_partition(model, "mypartition")
     table = db_introspection.get_partitioned_table(model._meta.db_table)
     assert len(table.partitions) == 0
+
+
```

Comment:
Current logic is to detach and delete partition right away, it may be useful to add an assertion if indeed deletion was executed correctly after detach.